src/main/webapp/app/course/course-scores/course-scores.component.ts

```diff
@@ -236,13 +236,9 @@ export class CourseScoresComponent implements OnInit, OnDestroy {
                 }
             }
         }
-        console.log(`Performed ${noOfComparisons} comparisons between old and new calculation method.`);
-        console.log(`Found ${noOfPointDifferencesFound} point differences between old and new calculation method.`);
-        console.log(`Found ${noOfScoreDifferencesFound} score differences between old and new calculation method.`);
```

Comment:
The `noOfPointDifferencesFound` variable is now unused.

Reply:
good catch, I removed those variables